docs/user/02-logs.md

```diff
@@ -0,0 +1,459 @@
+
+### Step 5: Rotate the Secret
+
+As used in the previous step, a Secret referenced with the **secretKeyRef** construct can be rotated manually or automatically. For automatic rotation, update the Secret's actual values and keep the Secret's keys stable. The LogPipeline watches the referenced Secrets and detects changes, so the Secret rotation takes immediate effect. When using a Secret owned by the [SAP BTP Operator](https://github.com/SAP/sap-btp-service-operator) you can configure a `credentialsRotationPolicy` with a specific `rotationFrequency` to achieve an automated rotation.
+
+### Step 6: Add a parser
+
+Typically, you want your logs shipped in a structured format so that a backend like [OpenSearch](https://opensearch.org/) can immediately index the content according to the log attributes. By default, a LogPipeline tries to parse all logs as a JSON document and enrich the record with the parsed attributes on the root record. Thus, logging in JSON format in the application results in structured log records. Sometimes, logging in JSON is not an option (the log configuration is not under your control), and the logs are in an unstructured or plain format. To adjust this, you can define your custom [parser](https://docs.fluentbit.io/manual/concepts/data-pipeline/parser) and activate it with a filter or a Pod annotation.
+
+The following example defines a parser named `dummy_test` using a dedicated `LogParser` resource type:
+
+```yaml
+kind: LogParser
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: dummy_test
+spec:
+  parser:
+    content: |
+      Format regex
+      Regex ^(?<INT>[^ ]+) (?<FLOAT>[^ ]+) (?<BOOL>[^ ]+) (?<STRING>.+)$
+```
+
+The parser is referenced by its name in a filter of the pipeline and is activated for all logs of the pipeline.
+
+```yaml
+kind: LogPipeline
+apiVersion: telemetry.kyma-project.io/v1alpha1
+metadata:
+  name: http-backend
+spec:
+  filters:
+    - custom: |
+        Name parser
+        Parser dummy_test
+  input:
+    ...
+  output:
+    ...
+```
+> **NOTE:** If you use a `custom` output, you put the LogPipeline in the [unsupported mode](#unsupported-mode).
+
+Instead of defining a filter, you can [annotate](https://docs.fluentbit.io/manual/pipeline/filters/kubernetes#kubernetes-annotations) your workload in the following way (here, the parser is activated only for the annotated workload):
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dummy
+  annotations:
+    fluentbit.io/parser: dummy_test
+spec:
+  ...
+```
+
+## Log record processing
+
+After a log record has been read, it is preprocessed by centrally configured plugins, like the `kubernetes` filter. Thus, when a record is ready to be processed by the sections defined in the LogPipeline definition, it has several attributes available for processing and shipment.
+
+![Flow](./assets/logging-flow.drawio.svg)
```

Comment:
2. There is already a numbering in place. Prefixing it with "Step" sounds not meaningful to me as that are not steps performed by users. Instead I prefixed it with "Stage" as the log record is going through that pipeline passing that stages.
1. Here we are more talking about steps of configuration and a diagram is not that meaningful in my POV. It would just repeat the headlines and brings no additional value..